pkg/plugin/driver/job/job.go

```diff
@@ -72,7 +72,7 @@ func NewPlugin(dfn plugin.Definition, namespace, sonobuoyImage, imagePullPolicy,
 // a Job only launches one pod, only one result type is expected.
 func (p *Plugin) ExpectedResults(nodes []v1.Node) []plugin.ExpectedResult {
 	return []plugin.ExpectedResult{
-		{ResultType: p.GetResultType()},
+		{ResultType: p.GetResultType(), NodeName: plugin.GlobalResult},
```

Comment:
Theoretically, if you have a node named "global" then it becomes ambiguous (when looking at the tarball by itself) if a job or daemonset was run.

However, that seems to be a fair tradeoff since by doing this we get rid of another ambiguity (if a job uploaded results with directories named with the nodes).

Both are unlikely and this is preferred since it helps make the two structures consistent.